### PR TITLE
Induction - Test Rule - Hugo Pernet

### PR DESCRIPTION
**Pull request by Freego1783**

### CODE REVIEW

File: rules/test-rule/rule.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+guid: df9552b7-1eff-4acc-9ae5-8e3db2d3cc82
+uri: test-rule
+created: 2023-01-23T15:23:10Z
+authors:
+  - title: Hugo Pernet
+    url: https://ssw.com.au/people/hugo-pernet
+---
+This is the introduction to my awesome rule.
+<!--endintro-->
+# Lists
```
Comment (tiagov8) [inline rules/test-rule/rule.md:13]: Hey @Freego1783 

please avoid making all headings h1 in the content...

https://webaim.org/projects/screenreadersurvey7/#heading

File: rules/test-rule/rule.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* Like
+* Lists
+
+1. But
+2. I
+3. Prefer
+4. Them
+5. Ordered
+
+#Link, Image and Video
```
Comment (tiagov8) [inline rules/test-rule/rule.md:25]: Missing space

File: rules/test-rule/rule.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Check out this [link](https://www.nintendo.fr/Jeux/Super-Nintendo/The-Legend-of-Zelda-A-Link-to-the-Past-841179.html "got it?"), it's clearly the best.
```
Comment (tiagov8) [inline rules/test-rule/rule.md:27]: Please follow https://ssw.com.au/rules/descriptive-links/

File: rules/test-rule/rule.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+::: img-small  
+![Cool image](test-img.jpg)
```
Comment (tiagov8) [inline rules/test-rule/rule.md:30]: Can we have a prefix as per https://ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-1-use-prefixes?

File: rules/test-rule/rule.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+:::
+
+Cool User Group teaser video:
+`youtube: https://www.youtube.com/embed/0akg1RqVQFo`
+
```
Comment (tiagov8) [inline rules/test-rule/rule.md:35]: Can you include a caption as per https://ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-use-the-titles-for-video-captions ?

File: rules/test-rule/rule.md (hunk 1 above)
Comment (Freego1783) [reply rules/test-rule/rule.md]: Done - changed to h3

File: rules/test-rule/rule.md (hunk 2 above)
Comment (Freego1783) [reply rules/test-rule/rule.md]: Fixed

File: rules/test-rule/rule.md (hunk 3 above)
Comment (Freego1783) [reply rules/test-rule/rule.md]: Done - Changed the whole thing. I wanted to make a silly joke but that was probably not appropriate.

File: rules/test-rule/rule.md (hunk 4 above)
Comment (Freego1783) [reply rules/test-rule/rule.md]: Done - added prefix

File: rules/test-rule/rule.md (hunk 5 above)
Comment (Freego1783) [reply rules/test-rule/rule.md:35]: Done - added a caption

**Comment by tiagov8** [comment]
Good work, closing as it's a test

**Pull request closed without merging**